src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -577,9 +565,6 @@ protected DCText inlineWord() {
         while (bp < buflen) {
             switch (ch) {
                 case '\n':
-                    newline = true;
-                    // fallthrough
-
                 case '\r': case '\f': case ' ': case '\t':
```

Comment:
This is not a big issue, but it seems that with #13343 adding the break to the `case '@'` below, the `@SuppressWarnings("fallthrough")` could now be removed from this method, right?

Reply:
The extra `@SuppressWarnings` has been removed.